Service/FindGithubRepositoryThroughOrganizationService.php

```diff
@@ -369,30 +329,35 @@ public function findGithubRepositoryThroughOrganizationHandler(?array $data = []
         $this->configuration = $configuration;
         $this->data = $data;
 
-        if ($organisationId) {
+        if ($organisationId !== null) {
```

Comment:
Again if $organisationId is not the exact same as !== null. These comments are here so we think twice before just changing these, isset or empty might be better/safer